src/fold/fold.rs

```diff
@@ -153,7 +153,7 @@ fn fold_file<T: Read>(mut file: BufReader<T>, bytes: bool, spaces: bool, width:
                                 match rfind_whitespace(slice) {
                                     Some(m) => {
                                         let routput = &slice[m + 1 .. slice.chars().count()];
-                                        let ncount = routput.chars().fold(0usize, |out, ch: char| {
+                                        let ncount = routput.chars().fold(0, |out, ch: char| {
```

Comment:
I haven't looked in detail, but given that on line there is a check `> 0` i guess this should have always been signed.